scheduler/src/cook/mesos/sandbox.clj

```diff
@@ -265,29 +265,87 @@
 
               :else
               (recur remaining-hostnames pending-sync-agent-send-performed))))))
-    {:error-handler (fn sandbox-publisher-error-handler [ex]
+    {:error-handler (fn start-host-sandbox-syncer-error-handler [ex]
                       (log/error ex "Sync of sandbox directories on pending hosts failed"))}))
 
+(defn- remove-task-ids-with-sandbox
+  "Filters the task-ids which have a sandbox directory from task-id->sandbox."
+  [datomic-conn task-id->sandbox]
+  (letfn [(retrieve-task-ids-with-sandbox [datomic-db task-ids]
```

Comment:
I thought we decided to favor `let` over `letfn`

Reply:
I undid the move of this function so that it doesn't appear in the diff. We can address this when we actually change the function.